src/MultichannelMarketing/GLAChannel.php

```diff
@@ -72,11 +72,20 @@ public function __construct( MerchantCenterService $merchant_center, AdsCampaign
 		$this->product_sync_stats = $product_sync_stats;
 		$this->campaign_types     = [];
 
-		if ( apply_filters( 'woocommerce_gla_enable_mcm', false ) === true ) {
+		if ( $this->is_enable_campaign_types() ) {
 			$this->campaign_types = $this->generate_campaign_types();
 		}
 	}
 
+	/**
+	 * Determines if the marketing channel campaigns types are enabled.
+	 *
+	 * @return bool
+	 */
+	protected function is_enable_campaign_types(): bool {
+		return apply_filters( 'woocommerce_gla_enable_mcm', false ) === true;
```

Comment:
💅 Since the filter name is `woocommerce_gla_enable_mcm`, I'd think `is_multichannel_marketing_enabled()` or just `is_mcm_enabled()` would be more suitable.

Reply:
Thanks for the suggestion! I've added here: [1968a2f](https://github.com/woocommerce/google-listings-and-ads/pull/2423/commits/1968a2f18a25f1dad2687489e1df764199d57758)